### DISCUSSION TITLE
Reminder: Epic Link and other related fields will be removed from Jira API and webhooks on May 31, 2022

### DISCUSSION THREAD

**Discussion by rocu-mapspeople** [pycontribs/jira#1283] [Ideas]
Any thoughts around this, will the python jira library receive any updates around this?

**Comment by adehad** [comment]
Definitely something we can look into if this affects the library.

Here is a link for those interesting in reading further:
https://community.developer.atlassian.com/t/deprecation-of-the-epic-link-parent-link-and-other-related-fields-in-rest-apis-and-webhooks/54048
